Google.PowerShell/Compute/GceCmdlet.cs

```diff
@@ -111,6 +124,66 @@ private void WriteWarnings(Operation op)
         }
 
         /// <summary>
+        /// Writes a progress record to give feedback to the user.
+        /// </summary>
+        /// <param name="op">The operation we are waiting on.</param>
+        /// <param name="progressMessage">The custom message to write to the progress bar. If null, this
+        ///  method will generate a message from the operation.</param>
+        private void WriteProgress(Operation op, string progressMessage)
+        {
+
+            int activityId = op.Id.GetHashCode();
+            string activity = progressMessage ?? op.Description;
+            if (activity == null)
+            {
+                string operationType = op.OperationType.Substring(0, 1).ToUpper()
```

Comment:
This code needs a comment, since it takes (a slow person like myself) > 0.1s to understand why.